int32 nLinear
int32 nAngular
int8 byResetOdometry
int8 byResetAngle
int8 byResetAlarm
